librz/main/rz-find.c

```diff
@@ -157,7 +160,9 @@ static int show_help(const char *argv0, int line) {
 		" -n         do not stop on read errors\n"
 		" -r         print using rizin commands\n"
 		" -s [str]   search for a specific string (can be used multiple times)\n"
-		" -S [str]   search for a specific wide string (can be used multiple times). Assumes str is UTF-8.\n"
+		" -w [str]   search for a specific wide string (can be used multiple times). Assumes str is UTF-8.\n"
+		" -I [str]   search for a specific function in import table.\n"
+		" -S [str]   search for a specific function in symbol table.\n"
```

Comment:
why just functions?

Reply:
This refers to the requirement from https://github.com/rizinorg/rizin/issues/2195

> Currently we have only magic, string, binary data search.
> It would be useful to be able to search for function calls in the set of binaries.
> 
> For example, it could look like this:
> 
> rz-find -I printf /usr/bin/ would search for printf() in executable file imports, and skip every non-executable file
> rz-find -S blaargh /usr/bin/ would search for the symbol blaargh() in the binary (can be import, export or just internal one), we could remove old -S option to search for wide strings and substitute it with -u (for Unicode).

Reply:
@XVilka ? I would just search for import/symbols no matter what. WDYT?

Reply:
Yes, no need to stick only to the functions, all imports/symbols are fine. It was a poor wording from my side.

Reply:
Ok, changed already.